piqueserver/scripts/runningman.py

```diff
@@ -61,18 +61,12 @@ def relink(connection):
 
 
 @command(admin_only=True)
+@target_player
 def unlink(connection, player=None):
```

Comment:
Player doesn't need to default to None here anymore either.

Reply:
Ah true, forgot to remove in a couple of places.